src/lib/models/query/Query.ts

```diff
@@ -5,10 +5,12 @@ export type ExpectedResultType =
   | 'COUNT'
   | 'CROSS_COUNT'
   | 'DATAFRAME'
+  | 'DATAFRAME_PFB'
   | 'DATAFRAME_TIMESERIES'
   | 'AGGREGATE_VCF_EXCERPT'
   | 'VCF_EXCERPT'
-  | 'VARIANT_COUNT_FOR_QUERY';
+  | 'VARIANT_COUNT_FOR_QUERY'
+  | undefined;
```

Comment:
idk about undefined

Reply:
So I have a variable of this type that I want to be undefined. What is the most elegant way to do this without adding undefined to the type? Ex:

```export let activeType: ExpectedResultType = undefined;```

Reply:
IntelliJ yells at me for doing that, but it seems to work so I'll remove `undefined` from the definition